hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/RocksDBStoreMBean.java

```diff
@@ -41,13 +49,21 @@
 /**
  * Adapter JMX bean to publish all the Rocksdb metrics.
  */
-public class RocksDBStoreMBean implements DynamicMBean {
+public class RocksDBStoreMBean implements DynamicMBean, MetricsSource {
 
```

Comment:
@elek  Will this automatically show up in the Promotheus ? or do we need to any extra plumbing ? See the comment in the JIRA where @jnp talks about Container metrics on Datanodes.

Reply:
@anuengineer I verified by checking the /prom servlet end point. I also made changes in the prometheus sink metric name sanitization code so that the RocksDB metric names don't look odd.